app/gateway/2.8.x/get-started/comprehensive/expose-services.md

```diff
@@ -288,17 +288,16 @@ A `201` message indicates the Route was created successfully.
 
 ## Verify the Route is forwarding requests to the Service
 
+By default, {{site.base_gateway}} handles proxy requests on port `8000`. This is often referred to as the Data Plane.
+
 {% navtabs %}
 {% navtab Using a Web Browser %}
 
-By default, {{site.base_gateway}} handles proxy requests on port `:8000`.
-
-From a web browser, enter `http://<admin-hostname>:8000/mock`.
+From a web browser, navigate to `http://<kong-gateway-host>:8000/mock/request`.
 
 {% endnavtab %}
-{% navtab Using the Admin API %}
 
-Using the Admin API, issue the following:
+{% navtab Using a command-line tool%}
```

Comment:
```suggestion
{% navtab Using the Admin API %}
```
Hold off on changing this. We have a lot of URL dependencies on this tab name, and it's currently a standard tab title we use across the docs, so we would have to do a sweep and change them all at once. If you think it should be changed, please file a ticket in DOCU and we'll take a look.

Reply:
Ack, yes it's not accurate to say that it's the Admin API.  I will file a ticket

Reply:
DOCU-2334